src/main/kotlin/pt/up/fe/ni/website/backend/model/Event.kt

```diff
@@ -2,13 +2,18 @@ package pt.up.fe.ni.website.backend.model
 
 import com.fasterxml.jackson.annotation.JsonProperty
 import jakarta.persistence.Entity
+import org.hibernate.validator.constraints.URL
 import java.util.Date
 
 @Entity
 class Event(
     title: String,
     description: String,
 
+    @JsonProperty(required = true)
+    @field:URL
+    var registerUrl: String,
```

Comment:
I don't think this should be required because we'll support past events as well (that is being done in #55 )

Reply:
I forgot to consider adding a past event. I've made changes so that registerUrl has the same properties as other links used, for example, in the Account model.